src/main/java/build/buildfarm/instance/shard/JedisClusterFactory.java

```diff
@@ -127,7 +132,7 @@ private static void deleteNodeExistingKeys(UnifiedJedis node) {
     ScanParams params = new ScanParams();
     params.match("*");
 
-    // get all of the keys for the particular node
+    // get all the keys for the particular node
```

Comment:
an odd choice? in Pittsburgh we typically only omit 'to be', not 'of'

Reply:
The LanguageTool plugin bundled with intellij was complaining about wordiness 🙃 https://languagetool.org/insights/post/wordiness/
I'll revert this and squash.